hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/action/compact/TestInlineCompaction.java

```diff
@@ -229,6 +229,9 @@ public void testSuccessfulCompactionBasedOnNumAndTime() throws Exception {
 
       // Then: ensure no compaction is executed since there are only 3 delta commits
       assertEquals(3, metaClient.getActiveTimeline().getWriteTimeline().countInstants());
+
+      // Ensure the elapsed time > INLINE_COMPACT_TIME_DELTA_SECONDS
+      Thread.sleep(60 * 1000);
```

Comment:
may I know why its flaky and why would this fix mitigate it. From what I see, we are setting trigger strategy to NUM_AND_TIME. 
why switching from 20 to 60 secs would work consistently ?

Reply:
I run this test on my laptop and get an assert exception
```
org.opentest4j.AssertionFailedError: Expect baseInstant to be less than or equal to latestDeltaCommit ==> 
Expected :true
Actual   :false
<Click to see difference>


	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:55)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
	at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:193)
	at org.apache.hudi.table.action.compact.CompactionTestBase.lambda$validateDeltaCommit$0(CompactionTestBase.java:103)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.apache.hudi.table.action.compact.CompactionTestBase.validateDeltaCommit(CompactionTestBase.java:95)
	at org.apache.hudi.table.action.compact.CompactionTestBase.runNextDeltaCommits(CompactionTestBase.java:148)
	at org.apache.hudi.table.action.compact.TestInlineCompaction.testSuccessfulCompactionBasedOnNumAndTime(TestInlineCompaction.java:227)
```

The original ut has two conditions: 
1. the elapsed time should less than 20s when the 3rd delta commits finishes. 
2. the elapsed time should greater than 20s when the 4th delta commits finishes. 

Actually, the elapsed time reaches 20s  when the 3rd delta commits finishes on my laptop, the compaction is executed earlier than expected.

So I increase the INLINE_COMPACT_TIME_DELTA_SECONDS  to ensure 3 delta commits can be finished in 60s. The sleep is to ensure compaction can be executed after the 4th commits.

Reply:
Another cause is the poor performance of my laptop, this is why this UT is successful on Azure too.